table/block_based/index_iterator.h

```diff
@@ -0,0 +1,131 @@
+//  Copyright (c) 2011-present, Facebook, Inc.  All rights reserved.
+//  This source code is licensed under both the GPLv2 (found in the
+//  COPYING file in the root directory) and Apache 2.0 License
+//  (found in the LICENSE.Apache file in the root directory).
+//
+// Copyright (c) 2011 The LevelDB Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file. See the AUTHORS file for names of contributors.
+
+#pragma once
+
+namespace ROCKSDB_NAMESPACE {
+
+class IndexIterator : public Cleanable {
```

Comment:
Is this a brand new class? Where is it used?

Reply:
Oh sorry. I added by mistake. Will remove it.